azure-pipelines-official.yml

```diff
@@ -14,35 +14,56 @@ jobs:
 
   variables:
     BuildPlatform: 'Any CPU'
-    InsertTargetBranchFullName: 'lab/d16.0stg'
-    InsertTargetBranchShortName: 'd16.0stg'
+    VisualStudio.MajorVersion: 16
+    VisualStudio.BranchName: 'lab/d16.0stg'
+    VisualStudio.ChannelName: 'int.d16.0stg'
+    VisualStudio.DropName: Products/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildNumber)
+
   steps:
+  - task: NuGetToolInstaller@0
+    inputs:
+      versionSpec: '4.9.2' 
```

Comment:
I've been trying to figure out how to do this for so long. Even asked the ADO team. How did you find this?

Reply:
On the internet :) 

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/nuget?view=vsts